stdlib/3/os/__init__.pyi

```diff
@@ -340,20 +340,18 @@ def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
                                                     List[AnyStr]]]: ...
 
 def abort() -> 'None': ...
-def execl(path: _PathType, arg0: Union[bytes, Text], *args: Union[bytes, Text]) -> None: ...
-def execle(path: _PathType, arg0: Union[bytes, Text],
-           *args: Any) -> None: ...  # Imprecise signature
-def execlp(path: _PathType, arg0: Union[bytes, Text], *args: Union[bytes, Text]) -> None: ...
-def execlpe(path: _PathType, arg0: Union[bytes, Text],
-            *args: Any) -> None: ...  # Imprecise signature
+def execl(path: _PathType, *args: Union[bytes, Text]) -> NoReturn: ...
```

Comment:
The first argument is actually called `file`, not `path` in both 2.7 and 3 (the documentation is wrong).